File: Java/src/br/com/dominio/s06/Exe05.java
package br.com.dominio.s06;

import java.util.Scanner;

public class Exe05 {

	public static void main(String[] args) {
		// Variaveis
		float p, m, exe;
		String e = "Excesso";
		Scanner teclado = new Scanner(System.in);
		
		// Entrada
		System.out.println("Insira o peso dos peixes: ");
		p = teclado.nextFloat();
		
		//processamento
		if(p > 50) {
			exe = (p - 50);
			m = (exe * 4.00f);
			System.out.printf(e + " de %.2f Kg e sua multa será de R$ %.2f ", exe, m);
		}else {
			exe= 0;
			m = 0;
			System.out.println("Quantidade dentro do limite");
			System.out.println(e + ": " + exe );
			System.out.println("Multa: " + m);
		}
		teclado.close();

	}

}

File: Python/Seção06/Exe05.py
#variaveis
excesso = 0
multa = 0
#entradas
peso = float(input("Informe a quantidade em kg de peixes: "))
#processamento
if peso > 50:
    excesso = peso - 50
    multa = excesso * 4.00
#saidas
    print("Seu excesso é de {0:.2f} Kg".format(excesso))
    print("Deverá pagar R$ {0:.2f} ".format(multa))
else:
    print("{0:.2f} Kg esta dentro do peso regulamentado".format(peso))
    print("Seu excesso é de {0:.2f} Kg".format(excesso))
    print("Deverá pagar R$ {0:.2f} ".format(multa))